earth_enterprise/rpms/shared/lib/key-value.sh

```diff
@@ -149,9 +149,12 @@ keyvalue_file_get()
     local _KFG_OUTPUT_VARIABLE_NAME="$3"
 
     if [ -f "$_KFG_STORE_PATH" ]; then
-        keyvalue_file_get_from_stdin \
-            "$_KFG_KEY" "$_KFG_OUTPUT_VARIABLE_NAME" \
-            < "$_KFG_STORE_PATH"
+        (
+            flock --shared -n 9 || exit 1
```

Comment:
I merged this with my unit tests (PR #576) and ran them and one is failing.
```
ASSERT:Get from file failed expected:<value2> but was:<>
```
It appears the output variable is being set but it's not escaping from the sub shell.

Reply:
Thanks, @tst-lsavoie. I was rushing this through without giving it enough attention. It should be fixed now.